packages/component/src/components/app-sidebar/app-updater-button/index.tsx

```diff
@@ -1,35 +1,111 @@
 import { useAFFiNEI18N } from '@affine/i18n/hooks';
-import { ResetIcon } from '@blocksuite/icons';
+import { CloseIcon, NewIcon, ResetIcon } from '@blocksuite/icons';
+import { assertExists } from '@blocksuite/store';
 import clsx from 'clsx';
+import { atom, useAtomValue, useSetAtom } from 'jotai';
+import { useEffect, useTransition } from 'react';
 
+import { changelogCheckedAtom, updateReadyAtom } from '../index.jotai';
 import * as styles from './index.css';
 
 interface AddPageButtonProps {
   className?: string;
   style?: React.CSSProperties;
 }
 
+const currentVersionAtom = atom(async () => {
+  const currentVersion = await window.apis?.updater.currentVersion();
```

Comment:
You assume this is a `no ssr` atom, but in another atom, you check no ssr. There may be consistent problems here.

Reply:
I think since this atom is async, ssr should be fine because aync atom in SSR will always block rendering right?

Reply:
actually no, suspense is not a guarantee of no ssr. Here it works just because upstream do the no ssr job.

The sever will response html with suspense in the first render if it did throw a promise(`onShellReady`)

Please read https://react.dev/reference/react-dom/server/renderToPipeableStream for more details